index.html

```diff
@@ -48,18 +48,25 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
           </div>
         </div>
         <div class="col-md-4">
-          <h2 class="heading">Contact</h2>
-          <form>
-            <div id="contact">
-              <label for="E-mail">Email</label>
-              <input type="email" placeholder="Enter your E-mail" id="E-mail" />
+          <div class="profile-section">
+            <div class="text-center">
+              <img src="./assets/images/uttam.jpg" alt="Shehroz Irfan" class="profile-image">
```

Comment:
Add your name in `alt` attribute.

Reply:
@uttam4124 
Excellent work. Just update the `alt` attribute wit your name and you're good to go!